.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
 script:
   - coverage run --source=pygrocy setup.py test
   - set -e
-  - pdoc --html pygrocy
+  - pdoc3 --html pygrocy
```

Comment:
is this needed?

Reply:
I don't why but with pdoc an error occurs about --html.
Maybe because it's python 3.x project.